app/scripts/metamask-controller.js

```diff
@@ -2609,7 +2646,7 @@ export default class MetamaskController extends EventEmitter {
       isUnlocked: this.isUnlocked(),
       accounts: await this.getPermittedAccounts(origin),
       ...this.getProviderNetworkState(
-        this.preferencesController.getUseRequestQueue() ? origin : undefined,
+        this.preferencesController?.getUseRequestQueue() ? origin : undefined,
```

Comment:
why is this needed now?

Reply:
not seeing this issue anymore so removing this for now: https://github.com/MetaMask/metamask-extension/commit/f2c2dbdb049f2e7268be45d684c8714acc1833f9